### ISSUE TITLE
Arguments to store_diagnostics_ciso_interior need a derived type

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#18]
In `marbl_ciso_mod::marbl_ciso_set_interior_forcing` the call to `marbl_store_diagnostics_ciso_interior` is preceded by the FIXME comment

```
the following arguments need to be grouped into a derived type
```

**Comment by mnlevy1981** [comment]
Similar comment applies to call to `marbl_store_diagnostics_ciso_surface_forcing`

**Comment by mvertens** [comment]
I cannot spend any more time on MARBL right now. We can leave this as a fix
me - or assign it to either you or Jon.

**Comment by mnlevy1981** [comment]
I don't think this is an urgent fix, I'm just trying to turn all the FIXMEs in the code into issue tickets so we don't lose track of them (I'm also including the issue number in the comments). I'll leave the assignee blank for now.

**Comment by mnlevy1981** [comment]
We may develop a more general solution (keeping #53 in mind); a possible solution may also involve returning CISO values to `marbl_mod` and letting `marbl_diagnostics_set_interior_forcing` handle calling `store_diagnostics_ciso`
